### PR TITLE
testcase/filesystem: Add errno verification and fix expect value of dprintf negative case function

### PR DESCRIPTION
**Pull request by KimDongEon**
Add errno verification and fix expect value of the "tc_libc_stdio_dprintf_invalid_fd_n()" function.

#This PR works properly when PR #4211 is merge

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ca6d84a6e77342eeebca7cdc45ceaca671d401b0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ca6d84a6e77342eeebca7cdc45ceaca671d401b0] - Code Rule Check OK.

File: apps/examples/testcase/le_tc/filesystem/fs_main.c [hunk 1]
```diff
@@ -3043,7 +3045,7 @@ static void tc_libc_stdio_dprintf_invalid_fd_n(void)
 
 	/* Testcase */
 	ret = dprintf(CONFIG_NFILE_DESCRIPTORS, "%s", str);
-	TC_ASSERT_EQ("dprintf", ret, 0);
+	TC_ASSERT_NEQ("dprintf", errno, 0);
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/filesystem/fs_main.c]: I think you'd better check with `EBADF`.

File: apps/examples/testcase/le_tc/filesystem/fs_main.c [hunk 2]
```diff
@@ -3018,8 +3018,10 @@ static void tc_libc_stdio_dprintf_p(void)
 	TC_ASSERT_GEQ("open", fd, 0);
 
 	/* Testcase */
+	errno = 0;
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/filesystem/fs_main.c]: How about using `set_errno` API?

**Comment by seinfra** [comment]
Target : [b2f70332b350f1d32f142c731abec62a715ce42e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b2f70332b350f1d32f142c731abec62a715ce42e] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [fc8e13436b34c3a25393b7eb3915f2dfcc255917] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [fc8e13436b34c3a25393b7eb3915f2dfcc255917] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6396b67df9df5bd755f283f9549558921abb744d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6396b67df9df5bd755f283f9549558921abb744d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [23a32ddb704aeeb6b43e810ae9af7cfc7155000f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [23a32ddb704aeeb6b43e810ae9af7cfc7155000f] - Code Rule Check OK.

File: apps/examples/testcase/le_tc/filesystem/fs_main.c (hunk 1 above)
Comment (KimDongEon) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: Okay. modify

File: apps/examples/testcase/le_tc/filesystem/fs_main.c (hunk 2 above)
Comment (KimDongEon) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: Remove errno check from the function.

**Pull request merged**